apps/system/js/statusbar.js

```diff
@@ -431,6 +440,12 @@
         case 'cardviewshown':
         case 'cardviewclosed':
           return 'cardview';
+        case 'appwillopen':
+        case 'stackchanged':
+          return 'appopening';
+        case 'appwillclose':
+        case 'stackchanged':
```

Comment:
and you'll want to add an entry in `_eventGroupState` otherwise it'll definitely not work